symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php

```diff
@@ -276,4 +277,16 @@ public function getConfigService()
         }
         return $this->configService;
     }
+
+    /**
+     * @return string
+     */
+    public function getVersion()
+    {
+        if (@include_once sfConfig::get('sf_root_dir') . "/../lib/confs/sysConf.php") {
+            $version = new sysConf();
+            $version = $version->getVersion();
+        }
+        return $version;
+    }
```

Comment:
I get this from OS existing code. needs to know if it is fine. This is regarding getting OS version